pkg/gofr/exporter.go

```diff
@@ -78,13 +78,13 @@ func (e *Exporter) processSpans(ctx context.Context, logger logging.Logger, span
 
 	resp, err := client.Do(req)
 	if err != nil {
-		logger.Error(err)
+		logger.Errorf("failed to create spans, error: %v", err)
```

Comment:
%w instead of %v seems better

Reply:
@ccoVeille we are not wrapping up the error here, just logging it. In my opinion, `%w` is specifically used for wrapping errors while `%v` is used for general formatting, including logging.

Reply:
Oh you are right, it's a logger line. I missed it